pyproject.toml

```diff
@@ -49,7 +49,7 @@ jupyter = [
     "mistune!=3.1.0",  # there is a bug in it
 ]
 zarr = [
-    "zarr>=2.16.0",
+    "zarr>=2.16.0,<3.0.0a0", # not yet compatible with 3.0.*
```

Comment:
We should mention somewhere that people can actually run zarr v3; they just won't be able to use anndata zarr streaming internals.

It's perfectly fine to use zarr v3 artifacts like I'm sure many people will do. But given this is an extra it might not be too confusing.

Reply:
I'm glad you put in the work long ago to separate this from the core package & dependencies, @Koncopd

Reply:
No, they actually can't use zarr v3, the imports will error. Actually i think i even should add some additional version check to avoid that if zarr v3 is installed already.

Reply:
Lets continue here https://github.com/laminlabs/lamindb/pull/2334